acquisitions/src/main/resources/karate-config.js

```diff
@@ -66,8 +66,17 @@ function fn() {
     createOrder: karate.read('classpath:thunderjet/mod-orders/reusable/create-order.feature'),
     createOrderLine: karate.read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature'),
     openOrder: read('classpath:thunderjet/mod-orders/reusable/open-order.feature'),
+    unopenOrder: read('classpath:thunderjet/mod-orders/reusable/unopen-order.feature'),
     closeOrder: read('classpath:thunderjet/mod-orders/reusable/close-order.feature'),
 
+    // invoices
+    createFiscalYear: read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature'),
```

Comment:
seems it already exists
https://github.com/folio-org/folio-integration-tests/pull/1542/files#diff-4b2c3abb8115ea123ed613d6527b535c050b703882ebb38b106cec5ac64435c9R42